### ISSUE TITLE
Module 01 notes

### ISSUE THREAD

**Issue by keith-kurak** [infinitered/cr-2024-intermediate-workshop-lessons#2]
### Higher priority / probably better that you make the edits
- I think we should go step by step through converting the first component (button seems like a good one). Break it into steps with small diffs. Then they can try the others on their own. For some folks, the lesson might only start to click here.
- The other components we don't walk through through, the expander should have the diff, since what actually changes is kind of a light touch. And if they try it on their own, they might miss one small part and want to figure it out. Also will be helpful for teachers troubleshooting :-D. That said, it might also be good to have the undiffed file on-hand for easy copying... might be a good use of the **files** folder - stuff the converted files in there.
- The changing of the styles on the Podcasts screen... many of those styles only have spacing in them, which we haven't changed. My understanding is that those components would still accept the original styles, but we're converting them so they would work with any future style changes we make. Is that correct? It'd be a good thing to call that out, so attendees understand why they still need to make all these edits to the screen even after they converted the components that the screen is using, especially since we don't do anything similar with profile, which has much fewer custom styles, admittedly. 
- On the font scaling part, let's remind them to try navigating to the different screens, noting what's wrong (just a "try it" after turning on font scaling would do I think)

### Lower priority: I could PR these later if you don't get to them
- Let's add a few of the "Try it" blurbs (see examples in 3/4/5). Sometimes people forget to stop copying, catch their breath, and actually run the code :-D
- Some additional "why" context for the first few steps in dark mode theming could be good. Like, "we're adding these files because we are creating an entirely separate theme".
- Podcasts screen diff: it's a really big diff that'd be nice to break up into 2 or 3 smaller diffs at logical points. That said, it's also not hard to follow once you're in it, so I could go either way on that. Maybe pre-return-statement, return statement, and styles?
- We could consider moving the code in steps that are copying in entire files to separate files in **files** folder. Don't feel strongly about it, though.

### Stuff I love
- Once the prep is done, I love turning them loose on updating the controls. Feels like an RPG when you get the boat or airship and can now travel anywhere.
- This is a really well-curated bite-sized chunk of an extremely realistic development scenario that definitely isn't getting done within an hour!
